pkg/elemental/elemental.go

```diff
@@ -432,6 +432,11 @@ func DeployImage(c v1.Config, img *v1.Image) error {
 	transientTree := strings.TrimSuffix(img.File, filepath.Ext(img.File)) + ".imgTree"
 	if img.Source.IsDir() {
 		transientTree = img.Source.Value()
+		err = DumpSource(c, transientTree, img.Source)
```

Comment:
DumpSource is not needed here as the source is already a directory. `DumpSource` just copies the source contents to the given directory (so for a dir source type it only rsyncs from one path to antoher)